aikau/src/main/resources/alfresco/layout/FixedHeaderFooter.js

```diff
@@ -234,12 +235,64 @@ define(["alfresco/core/ProcessWidgets",
             }
          ]);
 
+         // Add resize listeners to the header/footer
+         setTimeout(lang.hitch(this, this.addHeaderResizeListener));
```

Comment:
Also, I'm wondering if this should be protected by a boolean condition... I'm not sure that this will be needed by every widget that uses the mixin.